apollo/src/builder.ts

```diff
@@ -21,8 +21,8 @@ export const builder = new SchemaBuilder<{
 }>({
     plugins: [ScopeAuthPlugin],
     authScopes: async (context) => ({
-        anonymousRequest: !!!context.user.userId,
-        loggedIn: !!context.user.userId,
+        anonymousRequest: !!!context.user?.userId,
```

Comment:
this was behaving weird when I was passing in an invalid token. I need to spend more time making sure I'm doing things right, but it can wait until I do a proper second pass through auth and unjankify it all